tests/test_tabbed_content.py

```diff
@@ -209,7 +210,10 @@ def compose(self) -> ComposeResult:
         await tabbed_content.add_pane(TabPane("Added", id="new-1"), before="initial-1")
         assert tabbed_content.tab_count == 2
         assert tabbed_content.active == "initial-1"
-        assert [tab.id for tab in tabbed_content.query(Tab).results(Tab)] == [
+        assert [
+            ContentTab.sans_prefix(tab.id)
+            for tab in tabbed_content.query(Tab).results(Tab)
```

Comment:
I think we _used_ to need the `.results(Tab)` so that the type checker was happy but I think this is no longer the case.
I.e., I think you can remove `.results(Tab)` from here and the other lines below.

Reply:
Good catch; old habit.